pkg/converter/converter.go

```diff
@@ -132,32 +101,19 @@ func (cvt *LocalConverter) Convert(ctx context.Context, source string) error {
 
 	logger.Infof("converting image %s", source)
 	start = time.Now()
-	desc, err := cvt.driver.Convert(ctx, content)
+	desc, err := cvt.driver.Convert(ctx, cvt.provider)
 	if err != nil {
 		return errors.Wrap(err, "convert image")
 	}
-
```

Comment:
Why is this removed?

Reply:
We need to migrate to the new artifact reference spec, the annotations defined here are unnecessary anymore.